tools/cldr-apps/js/src/runGui.js

```diff
@@ -1,7 +1,7 @@
 import * as cldrGui from "./esm/cldrGui.js";
 
 function runGui() {
-  cldrGui.run();
+  return cldrGui.run();
 }
```

Comment:
A @return comment would make this less mysterious

Reply:
yes, or inline the whole thing into index.js